pkg/asset/ignition/bootstrap/bootstrap_in_place.go

```diff
@@ -0,0 +1,21 @@
+package bootstrap
+
+import (
+	"github.com/pkg/errors"
+
+	"github.com/openshift/installer/pkg/types"
+)
+
+// verifyBootstrapInPlace validate the number of control plane replica is one and that installation disk is set
+func verifyBootstrapInPlace(installConfig *types.InstallConfig) error {
+	if *installConfig.ControlPlane.Replicas != 1 {
+		return errors.Errorf("bootstrap in place requires a single control plane replica, current value: %d", *installConfig.ControlPlane.Replicas)
+	}
+	if installConfig.BootstrapInPlace == nil {
+		return errors.Errorf("missing bootstrap in place configuration")
```

Comment:
In other places in the validation code the validation framework is used to indicate exactly what field name is missing or has a bad value. Can that framework be used here, or do we need to update the error messages to be more specific? (See `pkg/types/baremetal/validation/platform.go` for an example)